package.json

```diff
@@ -16,7 +16,7 @@
   "author": "New York Media",
   "license": "MIT",
   "dependencies": {
-    "amphora-auth": "^1.0.0",
+    "amphora-auth": "git+ssh://git@github.com:clay/amphora-auth.git#subsite-support",
```

Comment:
will also want to make sure we bump amphora-auth before releasing this

Reply:
updated to ^1.2.0